src/DeterministicVI.jl

```diff
@@ -16,16 +16,23 @@ export ElboArgs
 
 """
 ElboArgs stores the arguments needed to evaluate the variational objective
-function
+function.
 """
 type ElboArgs{NumType <: Number}
     S::Int64
     N::Int64
+
+    # The number of components in the point spread function.
+    psf_K::Int64
```

Comment:
Isn't the number of psf components already known from the PSF itself? It's good to avoid declaring the same constant 2 places, so there's no potential for disagreement.